lib/streamlit/elements/lib/built_in_chart_utils.py

```diff
@@ -577,14 +584,39 @@ def _parse_y_columns(
     return y_column_list
 
 
+def _get_offset_encoding(
+    chart_type: ChartType,
+    stack: bool | ChartStackType | None,
+    color_column: str | None,
+) -> tuple[alt.XOffset, alt.YOffset]:
+    import altair as alt
```

Comment:
Also curious about this... I originally moved `altair` outside of the type check imports to the regular section and it failed python tests, so I followed existing pattern in this file of importing in the inner functions.

Reply:
Per discussion in standup, these are lazy-loaded specifically to avoid the import of `altair` unnecessarily as it improves initial load time of Streamlit (`import streamlit` )